site/pages/view-all-disruptions.test.tsx

```diff
@@ -68,87 +78,114 @@ const disruptions: TableDisruption[] = [
 
 const defaultNewDisruptionId = "acde070d-8c4c-4f0d-9d8a-162843c10333";
 
+const fetchSpy = vi.spyOn(global, "fetch");
+
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const useRouter = vi.spyOn(require("next/router"), "useRouter");
 beforeEach(() => {
     useRouter.mockImplementation(() => ({
         query: "",
     }));
+
+    vi.spyOn(global, "fetch");
```

Comment:
is this needed if we have fetchSpy above?

Reply:
resolved